src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: $cardFont;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__title {
+    margin-top: 40px;
```

Comment:
The `margin-top` value of `40px` seems too specific. Consider using a variable or a calculated value if this margin is supposed to be responsive to certain design criteria or is related to other elements' dimensions.